fintech-examples/fintech-server/src/main/java/de/adorsys/obpa/fintech/server/FinTechApplication.java

```diff
@@ -0,0 +1,16 @@
+package de.adorsys.obpa.fintech.server;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import de.adorsys.opba.fintechapi.config.EnableFinTechApi;
+
+@SpringBootApplication
```

Comment:
It is better from performance perspective to enlist exact packages to scan i.e.:
```
@SpringBootApplication(
        scanBasePackages = {
                "de.adorsys.opba.core.protocol.config",
                "de.adorsys.opba.core.protocol.controller",
                "de.adorsys.opba.core.protocol.service",
                "de.adorsys.opba.core.protocol.repository"
        }
)
```

Reply:
Scanning happens over @EnableFinTechApi
Added scanBasePackages to prevent purposeless scanning. @SpringBootApplication(scanBasePackages = {})